src/PKSim.Core/Model/Species.cs

```diff
@@ -1,39 +1,32 @@
 using System.Collections.Generic;
-using OSPSuite.Utility.Collections;
+using System.Linq;
 using OSPSuite.Core.Domain;
+using OSPSuite.Utility.Collections;
 
 namespace PKSim.Core.Model
 {
    public class Species : ObjectBase
    {
-      private readonly List<SpeciesPopulation> _allPopulations = new List<SpeciesPopulation>();
-      private readonly Cache<string, ParameterValueVersionCategory> _pvvCategories = new Cache<string, ParameterValueVersionCategory>(pvv => pvv.Name);
       public virtual string DisplayName { get; set; }
+
       public virtual bool IsHuman { get; set; }
 
-      public virtual SpeciesPopulation PopulationByName(string name)
-      {
-         return _allPopulations.FindByName(name);
-      }
+      private readonly List<SpeciesPopulation> _allPopulations = new List<SpeciesPopulation>();
+
+      private readonly Cache<string, ParameterValueVersionCategory> _pvvCategories = new Cache<string, ParameterValueVersionCategory>(pvv => pvv.Name);
+
+      public virtual SpeciesPopulation DefaultPopulation => _allPopulations.FirstOrDefault();
```

Comment:
Because of you comment, I have added the Default Population in Species. This is what is used when species is set in compound process